### PR TITLE
Follow up for VTOL transition additions

### PR DESCRIPTION
**Pull request by julianoes**
This follows up on https://github.com/dronecore/DroneCore/pull/150#issuecomment-344140095.

### CODE REVIEW

File: example/transition_vtol_fixed_wing/transition_vtol_fixed_wing.cpp [hunk 1]
```diff
@@ -106,6 +106,7 @@ int main(int /*argc*/, char ** /*argv*/)
     std::this_thread::sleep_for(std::chrono::seconds(10));
 
     // Change to VTOL
```
Comment (hamishwillee) [inline example/transition_vtol_fixed_wing/transition_vtol_fixed_wing.cpp]: Remove - incorrect and not needed.

Comment (julianoes) [reply example/transition_vtol_fixed_wing/transition_vtol_fixed_wing.cpp]: Thx will do.

**Comment by julianoes** [comment]
> What will happen on PX4 prior to version with ACK added? I assume that this will still timeout. Should there be a test for version?

It will timeout but still work, so not really a problem in my opinion.

> Comments on methods not quite correct - below for the transition to fixed wing:

Thanks, fixed.

**Comment by hamishwillee** [comment]
>> What will happen on PX4 prior to version with ACK added? I assume that this will still timeout. Should there be a test for version?

> It will timeout but still work, so not really a problem in my opinion.

I somewhat disagree, because you're getting a fail error even though the action will actually be executed (if the action was not being executed I'd feel differently).  The caller has to "magically" know that this fail is OK. 

Your call though. 

Otherwise all good.

**Comment by julianoes** [comment]
> I somewhat disagree, because you're getting a fail error even though the action will actually be executed (if the action was not being executed I'd feel differently). The caller has to "magically" know that this fail is OK.

DroneCore does not "officially" support any kind of release yet but is only tested against master. Therefore, we don't need to worry about earlier releases/versions at this point.

**Comment by hamishwillee** [comment]
Thanks!

**Pull request merged**